pkg/services/ngalert/api/tooling/definitions/contact_points.go

```diff
@@ -85,6 +85,18 @@ type LineIntegration struct {
 	Description *string `json:"description,omitempty" yaml:"description,omitempty" hcl:"description"`
 }
 
+type MqttIntegration struct {
+	DisableResolveMessage *bool `json:"-" yaml:"-" hcl:"disable_resolve_message"`
+
+	BrokerURL          *string `json:"brokerUrl,omitempty" yaml:"brokerUrl,omitempty" hcl:"broker_url"`
+	ClientID           *string `json:"clientId,omitempty" yaml:"clientId,omitempty" hcl:"client_id"`
+	Topic              *string `json:"topic,omitempty" yaml:"topic,omitempty" hcl:"topic"`
+	Message            *string `json:"message,omitempty" yaml:"message,omitempty" hcl:"message"`
+	Username           *string `json:"username,omitempty" yaml:"username,omitempty" hcl:"username"`
+	Password           *Secret `json:"password,omitempty" yaml:"password,omitempty" hcl:"password"`
+	InsecureSkipVerify *bool   `json:"insecureSkipVerify,omitempty" yaml:"insecureSkipVerify,omitempty" hcl:"insecure_skip_verify"`
```

Comment:
adding the ability to specify a custom CA cert was something that users also requested for the MQTT datasource